test/integration/instrumentation/promises/regressions.js

```diff
@@ -1,47 +0,0 @@
-/*
- * Copyright 2020 New Relic Corporation. All rights reserved.
- * SPDX-License-Identifier: Apache-2.0
- */
-
-'use strict'
-
-const helper = require('../../../lib/agent_helper')
```

Comment:
Seems like maybe we'd want to keep this coverage in case `when`/ libraries we are still using the old stuff for do something funky?

Reply:
Ah I see bluebird has a specific item around this. May still be nice to have something generic tho.

Reply:
i'm not sure. bluebird was the only place it was used before